src/account/AccountStorage.sol

```diff
@@ -32,14 +32,19 @@ struct ValidationData {
     bool isSignatureValidation;
     // How many execution hooks require the UO context.
     uint8 requireUOHookCount;
-    // The pre validation hooks for this function selector.
+    // The pre validation hooks for this validation function.
     FunctionReference[] preValidationHooks;
     // Permission hooks for this validation function.
     EnumerableSet.Bytes32Set permissionHooks;
     // The set of selectors that may be validated by this validation function.
     EnumerableSet.Bytes32Set selectors;
 }
 
+struct DirectCallValidationData {
+    bool allowed; // Whether or not this direct call is allowed.
+    FunctionReference[] preValidationHooks; // The set of pre validation hooks for this direct call.
+}
```

Comment:
Is this struct used, or can it be removed?

Reply:
Ah snap! It's slipped through the cracks from when I had custom storage for it! Removing!

Reply:
Addressed in [a09ffb2](https://github.com/erc6900/reference-implementation/pull/90/commits/a09ffb20ee47ad8ab7f2287882dfa754bb236fc3)